WordPress/Classes/Utility/Media/ImageLoader.swift

```diff
@@ -97,7 +97,7 @@
 
     /// Load an animated image from the given URL.
     ///
-    private func loadGif(with url: URL, from source: ImageSourceInformation) {
+    private func loadGif(with url: URL, from source: ImageSourceInformation, preferedSize size: CGSize) {
```

Comment:
Nitpick, typo: `prefered` -> `preferred`

Reply:
Thanks for noticing! There were quite a few `prefered` around. 😅